packages/cdktf-cli/src/bin/cmds/deploy.ts

```diff
@@ -76,6 +76,12 @@ class Command extends BaseCommand {
         // Setting value to negative will prevent it from being forwarded to terraform as an argument
         default: -1,
       })
+      .option("no-color", {
+        type: "boolean",
+        required: false,
+        desc: "Disables terminal formatting sequences in the output.",
+        default: false,
```

Comment:
In regards to our [docs](https://developer.hashicorp.com/terraform/cdktf/create-and-deploy/environment-variables) on the environment flag `FORCE_COLOR` we could change the default to
```suggestion
        default: process.env.FORCE_COLOR === "0",
```
which would disable colored output for Terraform, too, if the users sets the flag that we already document in our docs.

Reply:
Don't think the environment flag `FORCE_COLOR` can alter the colored output coming from Terraform. Looks like the `no-color` option needs to be passed to Terraform CLI as well.

Standard View with colored output:
<img width="753" alt="Screen Shot 2022-11-30 at 9 21 36 AM" src="https://user-images.githubusercontent.com/72527044/204820520-1ee05b23-2403-4610-85ce-8aa371eb785e.png">

View with `FORCE_COLOR=0`:
<img width="760" alt="Screen Shot 2022-11-30 at 9 22 08 AM" src="https://user-images.githubusercontent.com/72527044/204820808-454ac41b-2d09-4155-8d20-44099bc90818.png">

View with  `FORCE_COLOR=0` and `TF_CLI_ARGS === "-no-color"`:
<img width="751" alt="Screen Shot 2022-11-30 at 9 27 28 AM" src="https://user-images.githubusercontent.com/72527044/204821790-dada4662-fc6c-4f2c-8877-7054049a5d3d.png">

Main issue I see with this approach is overriding the `TF_CLI_ARGS` environment variable with a tag

Reply:
Yeah, `FORCE_COLOR` won't alter the output, but if the user sets it and that affects the default value of our `--no-color` option, the we'd act as if the user used that flag, although they used the env var 😅
I hope I'm not too bad at explaining what I try to say 😅

Reply:
Ahhhh I see what your saying now 😁